src/views/Manual/Manual.jsx

```diff
@@ -156,6 +156,7 @@ const Manual = () => {
 						<ResultsList
 							results={trialsPayload.trials}
 							resultsItemTitleLink={detailedViewPagePrettyUrlFormatter}
+							currentPage={Number(pager.page)}
```

Comment:
This is not really needed. You can do it as part of track.

Reply:
Done!